apps/condo/domains/subscription/schema/ServiceSubscription.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Generated by `createschema subscription.ServiceSubscription 'type:Text; isTrial:Checkbox; organization:Relationship:Organization:CASCADE; startAt:DateTimeUtc; finishAt:DateTimeUtc;'`
+ */
+
+const { Select, Checkbox, DateTimeUtc, Integer, Decimal } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/subscription/access/ServiceSubscription')
+const { ORGANIZATION_OWNED_FIELD } = require('../../../schema/_common')
+const { ServiceSubscription: ServiceSubscriptionAPI } = require('../utils/serverSchema')
+const get = require('lodash/get')
+const { OVERLAPPING_ERROR } = require('../constants/errors')
+
+
+const ServiceSubscription = new GQLListSchema('ServiceSubscription', {
+    schemaDoc: 'Availability time period of service features for client organization. Can be trial or payed.',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+
+        type: {
+            schemaDoc: 'System, from where subscription was created (our or external)',
+            type: Select,
+            options: 'default,sbbol',
+            isRequired: true,
+        },
+
+        isTrial: {
+            schemaDoc: 'Trial mode of subscription',
+            type: Checkbox,
+            isRequired: true,
+        },
+
+        organization: ORGANIZATION_OWNED_FIELD,
+
+        startAt: {
+            schemaDoc: 'When subscription was started',
+            type: DateTimeUtc,
+            isRequired: true,
+        },
+
+        finishAt: {
+            schemaDoc: 'When subscription should be ended',
+            type: DateTimeUtc,
+            isRequired: true,
+        },
+
+        unitsCount: {
+            schemaDoc: 'How much units are payed for this subscription',
+            type: Integer,
+        },
+
+        unitPrice: {
+            schemaDoc: 'How much one unit cost in Rubles for this subscription',
+            type: Decimal,
+            knexOptions: {
+                scale: 2,
+            },
+        },
+
+        totalPrice: {
+            schemaDoc: 'Total price of this subscription, calculated as unitCost * unitsPayed',
+            type: Decimal,
```

Comment:
Why not virtual?

Reply:
Maybe, it can be cases, that will need to query on this field.